data/adapters/dbSNFP_adapter.py

```diff
@@ -0,0 +1,158 @@
+    LABEL = 'dbSNFP_protein_variants'
+
+    SKIP_BIOCYPHER = True
+    OUTPUT_PATH = './parsed-data'
+    WRITE_THRESHOLD = 1000000
+
+    def __init__(self, filepath=None, collection='coding_variants', dry_run=True):
+        self.output_filepath = '{}/{}-{}-{}.json'.format(
+            DbSNFPAdapter.OUTPUT_PATH,
+            DbSNFPAdapter.LABEL,
+            collection,
+            filepath.split('/')[-1]
+        )
+
+        self.filepath = filepath
+        self.label = DbSNFPAdapter.LABEL
+        self.dataset = self.label
+        self.dry_run = dry_run
+        self.collection_name = collection
+
+        super(DbSNFPAdapter, self).__init__()
+
+    def process_file(self):
+        parsed_data_file = open(self.output_filepath, 'w')
+
+        record_count = 0
+
+        for line in open(self.filepath, 'r'):
+            if line.startswith('#chr'):
+                continue
+
+            data_line = line.strip().split('\t')
+
+            variant_id = build_variant_id(
+                data_line[0],
+                data_line[1],  # 1-based
+                data_line[2],
+                data_line[3]
+            )
+
+            protein_id = data_line[16].split('-')[0]
+            if ';' in protein_id:
+                protein_id = protein_id.split(';')[0]
+
+            # '.' is equivalent to None in this dataset
+            def data(pos):
+                return data_line[pos] if data_line[pos] != '.' else None
+
+            def long_data(pos):
+                try:
+                    return float(data_line[pos])
+                except:
+                    return None
+
+            gene_id = data(13)
+            if gene_id and ';' in gene_id:
+                gene_id = gene_id.split(';')[0]
+
+            transcript_id = data(14)
+            if transcript_id and ';' in transcript_id:
+                transcript_id = transcript_id.split(';')[0]
+
+            key = build_coding_variant_id(
+                variant_id, protein_id, transcript_id, gene_id)
+
+            if self.collection_name == 'variants_coding_variants':
+                to_json = {
+                    '_from': 'variants/' + variant_id,
+                    '_to': 'coding_variants/' + key,
+                    'source': 'dbSNFP 4.5a',
+                    'source_url': 'http://database.liulab.science/dbNSFP',
+
+                    'chr': data(0),
+                    'pos:long': long_data(1),
+                    'ref': data(2),  # 1-based
+                    'alt': data(3),
+                }
+            elif self.collection_name == 'coding_variants_proteins':
+                to_json = {
+                    '_from': 'coding_variants/' + key,
+                    '_to': 'proteins/' + protein_id,
+                    'source': 'dbSNFP 4.5a',
+                    'source_url': 'http://database.liulab.science/dbNSFP'
+                }
+            else:
+                to_json = {
+                    '_key': key,
+                    'ref': data(4),
```

Comment:
Why we don't add chr and pos here for coding variant?

Reply:
it was specified in the ticket: https://igvf.atlassian.net/browse/DSERV-272?focusedCommentId=28485